utils/economic-checks/src/util.rs

```diff
@@ -202,7 +202,7 @@ pub(crate) fn new_test_ext(
                 initial_authorities
                     .iter()
                     .cloned()
-                    .map(|(acc, _, _)| (acc, 1000)),
+                    .map(|(acc, _, _)| (acc, EXISTENTIAL_DEPOSIT * 2)),
```

Comment:
why x2?

Reply:
Just in case. These are validators' balances, that is in reality should be substantial. For the purpose of this application we just want to ensure the genesis block is applied successfully